x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/routes/entities/list.ts

```diff
@@ -53,7 +53,7 @@ export const listEntitiesRoute = (router: EntityAnalyticsRoutesDeps['router'], l
           const {
             page = 1,
             per_page: perPage = 10,
-            sort_field: sortField = 'entity.lastSeenTimestamp',
+            sort_field: sortField = 'entity.last_seen_timestamp',
```

Comment:
After a text search inside the security solution, I've found three more places referencing this variable.

https://github.com/klacabane/kibana/blob/167-rename-entity-fields/x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/routes/entities/list.ts#L56

https://github.com/klacabane/kibana/blob/167-rename-entity-fields/x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/elasticsearch_assets/ingest_processor_steps/remove_entity_definition_fields_step.ts#L18

https://github.com/klacabane/kibana/blob/167-rename-entity-fields/x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/elasticsearch_assets/ingest_pipeline.ts#L70


I also found references to:  `entity.schemaVersion`, `entity.definitionVersion`, `entity.identityFields`, `entity.definitionId`, `entity.displayName`, `entity.firstSeenTimestamp`. Are these also changing?

Reply:
Thanks @machadoum! I've renamed found occurrences in https://github.com/elastic/kibana/pull/195895/commits/5cbe4b73d4dfce4d1918a4fad08d79e44692b2a9. The other references are from the entity definition object which did not change